src/components/v2v3/V2V3Project/V2V3ProjectSettings/pages/ProjectUpgradesPage/ProjectUpgradesPage.tsx

```diff
@@ -22,16 +28,35 @@ export function ProjectUpgradesPage() {
         Good enuf for now.
          */}
         <Statistic title="Current version" value={`V${cv}`} />
+
         {!isUpgradeAvailable && (
           <p>
             <Trans>Your project is up to date!</Trans>
           </p>
         )}
 
         {isUpgradeAvailable && (
-          <Button type="primary" onClick={() => setUpgradeWizardOpen(true)}>
-            <Trans>Start upgrade</Trans>
-          </Button>
+          <>
+            <h3>
+              <Trans>Available upgrades</Trans>
+            </h3>
+            {availableUpgrades?.map(v => {
+              const upgrade = VERSIONS[v]
+              return (
+                <div key={v}>
+                  <h4>{upgrade.name}</h4>
+                  {upgrade.description && <p>{upgrade.description()}</p>}
+                  <Button
+                    size="small"
+                    type="primary"
+                    onClick={() => setUpgradeWizardOpen(v)}
+                  >
+                    <Trans>Upgrade to {upgrade.name}</Trans>
+                  </Button>
+                </div>
+              )
+            })}
```

Comment:
Each upgrade will get a row that looks like this.

<img width="755" alt="Screenshot 2023-02-14 at 2 11 14 PM" src="https://user-images.githubusercontent.com/12551741/218633542-e18f8e11-48ea-49a7-8b85-3e8711fbfe64.png">